mmv1/products/alloydb/Cluster.yaml

```diff
@@ -256,9 +256,9 @@ properties:
       An object containing a list of "key": value pairs. Example: { "name": "wrench", "mass": "1.3kg", "count": "3" }.
   - !ruby/object:Api::Type::String
     name: 'databaseVersion'
-    output: true
+    default_from_api: true
     description: |
-      The database engine major version. This is an output-only field and it's populated at the Cluster creation time. This field cannot be changed after cluster creation.
+      The database engine major version. This is an optional field and it's populated at the Cluster creation time. This field cannot be changed after cluster creation.
```

Comment:
If this field cannot be changed, could we add `immutable:true` to it?

Reply:
1. IIUC, an immutable field will cause destruction (and recreation) of the existing cluster if the field value is change in Terraform, which is highly undesirable.
2. We will have the ability to change this field in the near future.

Reply:
Thanks for the explanation, LGTM.